opentech/apply/projects/models.py

```diff
@@ -5,6 +5,36 @@
 from django.utils.translation import ugettext as _
 
 
+APPROVAL_CHOICES = [
+    ('approved', 'Approved'),
+    ('rejected', 'Rejected'),
+]
+
+
+class Approval(models.Model):
+    project = models.ForeignKey("Project", on_delete=models.CASCADE)
+    by = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+
+    status = models.TextField(default='', choices=APPROVAL_CHOICES)
```

Comment:
Do we need a status on the approval? Either it is approved or not and we aren't tracking non approvals except in the logs.